osu.Game/Overlays/BeatmapListing/BeatmapListingSearchControl.cs

```diff
@@ -42,6 +43,8 @@ public class BeatmapListingSearchControl : CompositeDrawable
 
         public Bindable<SearchPlayed> Played => playedFilter.Current;
 
+        public Bindable<SearchExplicit> Explicit => explicitFilter.Current;
```

Comment:
In this change, you've used the terms:

`SearchExplicit`
`AllowExplicitContent`
`Explicit`
`HasExplicitContent `

My main issue is the config variable, which doesn't really suggest what the setting does (would prefer `BypassExplicitContentWarning` if that's the intended purpose).

I'll go through and rename some of these.

Reply:
I've used the term `AllowExplicitContent` because not only is it meant for bypassing explicit content warning, but it is also meant for unblocking actual explicit content outside the "beatmap info" overlay.

i.e. in web, here's how it looks with the global setting disabled:
![image](https://user-images.githubusercontent.com/22781491/104843673-e603f300-58dc-11eb-9bd0-5f2565b2e674.png)

and here's how it looks with the setting enabled:
![image](https://user-images.githubusercontent.com/22781491/104843723-23688080-58dd-11eb-8f53-fa488c99a31b.png)

Reply:
Then how about `ShowOnlineExplicitContent`? Because it
- doesn't apply to local content (and shouldn't)
- it's not disallowing; it's hiding by default